### DISCUSSION TITLE
Info about fetching frontend assets from a custom resource

### DISCUSSION THREAD

**Discussion by hsanand61564** [iTwin/itwinjs-core#1313] [Q&A] [answered]
Hi

I am using the itwin-viewer-react as my viewer, so I modified the base for the resources using the below

![image](https://user-images.githubusercontent.com/60655118/116864952-533a5a80-ac26-11eb-8664-2e30051646aa.png)

Is there a way where we can change the resource path for all images and cursors like the json files in the above case.

![image](https://user-images.githubusercontent.com/60655118/116865014-736a1980-ac26-11eb-94f2-82b7167fffa6.png)

**Comment by pmconne** [comment]
@wgoehrig?

**Comment by aruniverse** [comment, marked as answer]
As far as I know, at the moment we don't currently support this. Internally we've discussed this a bit, and its on our backlog. The enhancement hasn't been finalized yet, and we're not sure when we'll get to this, but we are thinking about it. Will keep you posted if there's any updates.
